Packs/Bitbucket/Integrations/Bitbucket/Bitbucket.py

```diff
@@ -0,0 +1,1479 @@
+    elif author_name and not author_email:
+        raise Exception('Please enter an email as well.')
+    elif not author_name and author_email:
+        raise Exception('Please enter a name as well.')
+    client.commit_create_request(body, repo)
+    return CommandResults(readable_output='The commit was created successfully.')
+
+
+def commit_list_command(client: Client, args: Dict) -> CommandResults:
+    """ Returns a list of all the commits according to the given params.
+    Args:
+        client: A Bitbucket client.
+        args: Demisto args.
+    Returns:
+        A CommandResult object with the list of commits.
+    """
+    repo = args.get('repo', client.repository)
+    file_path = args.get('file_path', None)
+    excluded_branches = args.get('excluded_branches', None)
+    included_branches = args.get('included_branches', None)
+    arg_limit = arg_to_number(args.get('limit'))
+    limit = arg_limit if arg_limit else 50
+    page = arg_to_number(args.get('page', 1))
+    page_size = min(limit, 100)
+    excluded_list = argToList(arg=excluded_branches)
+    included_list = argToList(arg=included_branches)
+    response = client.commit_list_request(repo, file_path, page, page_size, excluded_list, included_list)
+    results = check_pagination(client, response, limit)
+    human_readable = []
+    key_list = []
+    for value in results:
+        d = {'Author': value.get('author').get('raw'),
+             'Commit': value.get('hash'),
+             'Message': value.get('message'),
+             'CreatedAt': value.get('date')}
+        human_readable.append(d)
+        key_list.append(value.get('hash'))
+
+    headers = ['Author', 'Commit', 'Message', 'CreatedAt']
+    readable_output = tableToMarkdown(
+        name='The list of commits',
+        t=human_readable,
+        removeNull=True,
+        headers=headers
+    )
+    return CommandResults(
+        readable_output=readable_output,
+        outputs_prefix='Bitbucket.Commit',
+        outputs=results,
+        raw_response=results,
+        outputs_key_field=key_list
+    )
+
+
+def file_delete_command(client: Client, args: Dict) -> CommandResults:
+    """ Deletes a file in Bitbucket.
+    Args:
+        client: A Bitbucket client.
+        args: Demisto args.
+    Returns:
+        A CommandResult object with a success message, in case of a successful action.
+    """
+    repo = args.get('repo', client.repository)
+    message = args.get('message', '')
+    branch = args.get('branch', '')
+    file_name = args.get('file_name', '')
+    author_name = args.get('author_name', None)
+    author_email = args.get('author_email', None)
+    body = {
+        'message': message,
+        'branch': branch,
+        'files': file_name
+    }
+    if author_name and author_email:
+        body['author'] = f'{author_name} <{author_email}>'
+    elif author_name and not author_email:
+        raise Exception('Please enter an email as well.')
+    elif not author_name and author_email:
```

Comment:
Same as the above
You can do something like:
elif not author_email or not author name:
raise Exception

Reply:
But in case that both of them does not exist, it is ok, bitbucket will use the default user, the one that is connected to bitbucket and making the request.
Only if one of these parameters are missing we need to raise exception. 
If we will do "elif not author_email or not author name:" it will be true also in the case above, and it is not correct.

Reply:
ok. So let's still do it in on elif -
`if not author_email and author_name or not author_name and author_email:`
raise Exception('Both author email and author name should be given')